sensor_msgs/Image image
---
ImageFeatureMeasurement[] features
